package.json

```diff
@@ -61,5 +63,10 @@
     "@storybook/preset-create-react-app": "^2.1.1",
     "@storybook/react": "^5.3.17",
     "prettier": "^2.0.2"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm run prettier"
```

Comment:
I'm too late as it's already been merged, but should we run a command that changes files in a `pre-commit`? Shouldn't it only run checks? @turbaszek @pastDexter

Reply:
Yup. Running this command doesn't prevent you from making a commit. All files are committed with wrong formatting and corrected files are left unstaged

Reply:
https://prettier.io/docs/en/precommit.html#option-2-pretty-quickhttpsgithubcomazzpretty-quick ?

Reply:
:O you are right

Reply:
Too much moving parts... does it mean that we need install `pretty-quick`?

Reply:
It's quite interesting that prettier doesn't rise error when files were staged

Reply:
`pretty-quick` seems to handle adding updated files to the commit and it only runs on changed files, which should be much faster than it is now

Reply:
Why would it raise any error? It's a perfectly valid situation

Reply:
All git commits hooks I know (and we have over 20 in Airflow) fails when they change file.